app/helpers/employee_helper.rb

```diff
@@ -0,0 +1,2 @@
+module EmployeeHelper
+end
```

Comment:
If you don't occupy the helper, you can delete it.

Reply:
Deleted employee_helper.rb.